src/Page.js

```diff
@@ -344,6 +348,31 @@ Page.prototype.concatenateHeadingsAndKeywords = function () {
   });
 };
 
+/**
+ * Filters out elements on the page based on config tags
+ * @param tags to filter
+ * @param content of the page
+ */
+Page.prototype.filterTags = function (tags, content) {
+  if (tags === undefined) {
+    return content;
+  }
+  if (!_.isArray(tags)) {
+    return this.filterTags([tags], content);
+  }
```

Comment:
- use `Array.isArray`.
- use a variable instead of recursion? similar to `Site.prototype._rebuildAffectedSourceFiles`.
  ```js
  const tagsArray = Array.isArray(tags) ? tags : [tags];
  ```